### ISSUE TITLE
Trying to use video compression

### ISSUE THREAD

**Issue by Richi2293** [PatrissolJuns/react-native-audio-video-tools#5]
Hi,

sorry if I'm opening the second issue today .

However I have successfully installed the library in these steps:
- install react-native-audio-video-tools
- install react-native-ffmpeg
- added in pod file:
`pod 'react-native-ffmpeg/full-gpl', :podspec => '../node_modules/react-native-ffmpeg/react-native-ffmpeg.podspec'`
- everything starts successfully 🥳 

My problem is that when I go to get a file and try to compress, I have this output in the logs:

![image](https://user-images.githubusercontent.com/5848941/107795280-c1266280-6d58-11eb-858a-bfed41453e94.png)

For what strange reason?

-----------
iOS 14.4
ReactNative 0.63.4

**Comment by PatrissolJuns** [comment]
Did you well follow the installation steps of react-native-ffmpeg ? Especially Packages step

**Comment by Richi2293** [comment]
Yes I added the package in the podfile and started the "pod install" command, just that right?
(I wrote it in the comment above)

**Comment by Richi2293** [comment]
I think I have found the problem, it just doesn't work with the latest version of react-native-ffmpeg (0.5.1)

**Comment by PatrissolJuns** [comment]
@Richi2293 Nice to hear that
